jenkins/build.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+###
+### NOTE: Replacement for hack/jenkins/build.sh ???
```

Comment:
I agree, but we are holding off on bringing code from `hack/jenkins` to this repository until we have CI set up for them. Currently, `hack/jenkins/build.sh` gets hit by PR Jenkins so at least we have a little bit of signal that it works.